bugshaker/src/main/java/com/github/stkent/bugshaker/ScreenshotUtil.java

```diff
@@ -0,0 +1,49 @@
+package com.github.stkent.bugshaker;
+
+import java.io.File;
+
+import android.app.Application;
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+import com.github.stkent.bugshaker.flow.email.screenshot.BasicScreenShotProvider;
+import com.github.stkent.bugshaker.flow.email.screenshot.ScreenshotProvider;
+import com.github.stkent.bugshaker.flow.email.screenshot.maps.MapScreenshotProvider;
+import com.github.stkent.bugshaker.utilities.Logger;
+
+
+public class ScreenshotUtil {
+
+	private static final String SCREENSHOTS_DIRECTORY_NAME = "bug-reports";
+	private static final String SCREENSHOT_FILE_NAME = "latest-screenshot.jpg";
+	public static final String IMAGE_FILE_NAME = "Screenshot.png";
```

Comment:
Yeah, so we should change the name of these variables accordingly so that they represent what they are. Like "ANNOTATED_SCREENSHOT_FILE_NAME" and "DEFAULT_SCREENSHOT_FILE_NAME"